typescript/testsuite/execution_impl/example_testsuite_configurator.ts

```diff
@@ -0,0 +1,48 @@
+import { ExampleTestsuite } from "../testsuite_impl/example_testsuite";
+import { TestSuite } from "kurtosis-testsuite-api-lib"; //TODO (Ali)
+import * as log from "loglevel";
+import { Result, err, ok } from "neverthrow";
+import { ExampleTestsuiteArgs } from "./example_testsuite_args";
+
+
+class ExampleTestsuiteConfigurator {
+	
+	construcor () {}
+	
+	public setLogLevel(logLevelStr: string): Result<null, Error> {
+		const newLog: log.Logger = log.getLogger(logLevelStr);
+		newLog.setLevel(newLog.getLevel());
```

Comment:
This won't actually do what you want - it will create a new logger object with the level, yes... but then all the calls  in the code use the global logger, which will be unaffected.

The right way for `loglevel` (and the reason we chose to use `loglevel` over `winston`) appears to be `log.setLevel` on the global object, to set the global logger's level